### PR TITLE
dbeaver-community: update to 7.2.1

### PR DESCRIPTION
**Pull request by lhaeger**
###### Tested on

macOS 10.15.6 19G2021
Xcode 12.0 12A7209

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: databases/dbeaver-community/Portfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -* vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```
[the remark this answers is not in the document]
Comment (lhaeger) [reply databases/dbeaver-community/Portfile:1]: Good catch, that was indeed unintended.

File: databases/dbeaver-community/Portfile [hunk 2]
```diff
@@ -27,9 +27,20 @@ long_description            Free multi-platform database tool for developers, SQ
...
-checksums                   rmd160  cbb9ebebc75c4eb953f30eb9cc97f9d65da7f13c \
-                            sha256  cd38a2f96e400527f70dc12d3dd74baa89e5276b616644d4a0a03796402e7dcc \
-                            size    60295252
+checksums                   rmd160  10264039eddb00ef4d3ddd54653fceb282be8d29 \
+                            sha256  78c1152ea64a38d1e895e45da0e1bd8a25bf784b8b82bde53a493ea4b56676ce \
+                            size    60339319
+
+# Mark this port as re-distributing a pre-built binary rather than building from source
+variant                     prebuilt \
+    description             {Installs the pre-built binary available from} ${homepage}. {}
```
[the remark this answers is not in the document]
Comment (lhaeger) [reply databases/dbeaver-community/Portfile:36]: Also looks better, I agree.

File: databases/dbeaver-community/Portfile [hunk 3]
```diff
@@ -27,9 +27,20 @@ long_description            Free multi-platform database tool for developers, SQ
...
+default_variants            +prebuilt
```
[the remark this answers is not in the document]
Comment (lhaeger) [reply databases/dbeaver-community/Portfile:37]: The discussion kind of ended in silence and I was hoping to revive it by this example here. Thanks for picking it up...

File: databases/dbeaver-community/Portfile [hunk 4]
```diff
@@ -27,9 +27,20 @@ long_description            Free multi-platform database tool for developers, SQ
...
+
+# Until building from source has been added, prevent deselecting the prebuilt variant
+if { ! [variant_isset "prebuilt"] } {
+    known_fail yes
+    return -code error "${name} currently cannot be build from source and requires using the default \"+prebuilt\" variant."
```
[the remark this answers is not in the document]
Comment (lhaeger) [reply databases/dbeaver-community/Portfile:42]: I've had it in the pre-fetch phase earlier and obviously did not fully test everything after moving it into the global part, sorry for that. Will fix this as well.

File: databases/dbeaver-community/Portfile (hunk 3 above)
[the remark this answers is not in the document]
Comment (kencu) [reply databases/dbeaver-community/Portfile:37]: It did not end in **silence**. 

And we certainly did not agree on this, although @lhaeger was very outspoken that this was what he wanted to have happen.

Actually - we did not agree that this entire process would go ahead at all, and there was considerable opposition.

[the remark this answers is not in the document]
Comment (kencu) [reply databases/dbeaver-community/Portfile:37]: Having a variant to indicate a non-buildable version of the port has a lot of issues -- for one, it is not a "variant". For another, any other variants become ridiculous, as the port is coming from a binary. There are (IMHO) more clear methods of doing this that we will not have to come back later an undo. I suggested it be indicated in the name of the port that it was installing a binary. Chris said it should be a category of ports installing a binary with no indication in the name. Only @lhaeger (as far as I remember) wanted the indication in the port as a "variant".

**Comment by kencu** [comment]
This should not be committed without an actual plan for this process being agreed upon in the mailing lists.

**Comment by lhaeger** [comment]
Closing due to next version update already available...

**Pull request closed without merging**